Tests/Tests/Encryption/Keychain/CDTEncryptionKeychainManagerTests.m

```diff
@@ -0,0 +1,337 @@
+//
+//  CDTEncryptionKeychainManagerTests.m
+//  Tests
+//
+//  Created by Enrique de la Torre Fernandez on 15/04/2015.
+//  Copyright (c) 2015 IBM Cloudant. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "CDTEncryptionKeychainManager+Internal.h"
+
+#import "CDTEncryptionKeychainUtils.h"
+#import "CDTEncryptionKeychainConstants.h"
+
+#import "CDTMockEncryptionKeychainStorage.h"
+
+#define CDTENCRYPTIONKEYCHAINMANAGERTESTS_PASSWORD_DEFAULT @"password"
```

Comment:
Okay, perhaps add a comment above the test default password here. Otherwise it does look over-simplistic and it's not worth someone else asking the question later :)